src/composables/useNavigationGuards.ts

```diff
@@ -13,6 +17,53 @@ export default function useNavigationGuards() {
   const { setShowRedirectModal, isVeBalSupported } = useVeBal();
   const { setSidebarOpen } = useSidebar();
 
+  router.beforeEach((to, from, next) => {
+    const betaEnv =
+      window.location.host.split('.')[0] === 'beta' ||
+      window.location.host.split('.')[0] === 'staging';
+    const subdomain = betaEnv
+      ? window.location.host.split('.')[1]
+      : window.location.host.split('.')[0];
+    const networkFromSubdomain = networkFromSlug(subdomain);
+    if (networkFromSubdomain) {
+      // old format subdomain redirect
+      const networkFromUrl = networkFromSlug(to.params.networkSlug?.toString());
+      localStorage.setItem(
+        'networkId',
+        (networkFromUrl || networkFromSubdomain).toString()
+      );
+      window.location.href = networkFromUrl
+        ? `${appUrl()}${to.fullPath}`
+        : `${appUrl()}/${config[networkFromUrl || networkFromSubdomain].slug}${
+            to.fullPath
+          }`;
```

Comment:
This should be a testable function that returns a URL. Please write some tests so we can be sure a variety of different input URLs will be redirected to the right place.

Reply:
👍 Refactored the code to split into separate functions and added unit tests. 

Separately, I don't think file `/src/lib/config/test.json` is used for anything - can I delete it?